src/DOM.js

```diff
@@ -2199,7 +2199,9 @@ class HTMLIFrameElement extends HTMLSrcableElement {
                 const parentWindow = this.ownerDocument.defaultView;
                 const options = parentWindow[symbols.optionsSymbol];
 
-                url = _normalizeUrl(url, options.baseUrl);
+                url = _normalizeUrl(res.url, options.baseUrl);
```

Comment:
note: technically this line is all that's needed for resources within the redirected AFRAME url to load correctly (the other changes to do with making sure `iframe.src` has the correct URL afterwards)